src/LoginPage.cpp

```diff
@@ -131,6 +137,7 @@ LoginPage::LoginPage(QWidget *parent)
         serverLayout_ = new QHBoxLayout();
         serverLayout_->addWidget(serverInput_, 0, Qt::AlignVCenter);
 
+        form_layout_->addLayout(matrixid_error_layout_);
```

Comment:
I think it makes more sense to have the error below the matrix id input.

Reply:
For me looks strange, because it is lots of space between login and password, but if it's not a problem, I'm agree

Reply:
The label should probably just be hidden, if it is empty maybe?

Reply:
Yea you're right. (sorry I have deleted last comment)